Jenkinsfile

```diff
@@ -75,9 +75,10 @@ catchError() {
                 checkout scm
                 
                 sh """set -x
-                      git submodule foreach "git reset --hard || true" || true
-                      git reset --hard || true
-                      git submodule update --init || true
+                      git reset --hard
+					  git submodule foreach git clean --force
```

Comment:
bitte einrückung noch korrigieren

Reply:
Done. Also added VERBOSE=1 to make to see what is failing there (osx)